core/src/test/java/org/arquillian/smart/testing/configuration/ObjectMapperTest.java

```diff
@@ -226,17 +244,22 @@ public void setDummyObject(DummyObject dummyObject) {
             this.dummyObject = dummyObject;
         }
 
+        public void setMultiple(String[] multiple) {
+            this.multiple = multiple;
+        }
+
         @Override
         public List<ConfigurationItem> registerConfigurationItems() {
             List<ConfigurationItem> configItems = new ArrayList<>();
             configItems.add(new ConfigurationItem("i", null, 0));
             configItems.add(new ConfigurationItem("s"));
-            configItems.add(new ConfigurationItem("d", null, 0.0));
-            configItems.add(new ConfigurationItem("b", null, false));
-            configItems.add(new ConfigurationItem("l", null, Collections.EMPTY_LIST));
-            configItems.add(new ConfigurationItem("m", null, Collections.EMPTY_MAP));
-            configItems.add(new ConfigurationItem("as", null, new String[0]));
-            configItems.add(new ConfigurationItem("e", null, TestEnum.FOO));
+            configItems.add(new ConfigurationItem("d", 0.0));
+            configItems.add(new ConfigurationItem("b", false));
+            configItems.add(new ConfigurationItem("l", Collections.EMPTY_LIST));
+            configItems.add(new ConfigurationItem("m", Collections.EMPTY_MAP));
+            configItems.add(new ConfigurationItem("as", new String[0]));
+            configItems.add(new ConfigurationItem("e", TestEnum.FOO));
+            configItems.add(new ConfigurationItem("multiple", "my.property.*"));
 
```

Comment:
@lordofthejars : I am not sure Why @MatousJobanek Removed constructor ` ConfigurationItem(String paramName, Object defaultValue)`. You can check it [here](https://github.com/arquillian/smart-testing/commit/2f59136f1e8114df1e373e5deab06296412e652e)

Maybe he has some good reason.

Reply:
this is not the default value, but the second parameter is the system property name

Reply:
Yeah, I had good reason for it @dipak-pawar.
Having the constructor: 
`ConfigurationItem(String paramName, Object defaultValue)`
and at the same time also
`ConfigurationItem(String paramName, String systemProperty)`
may lead to confusions and, in some cases, also to a wrong usage of it. In the original stage of this PR, there was exactly the case of the wrong usage of the constructors. 
In addition, I don't see it as a big benefit to have the constructor there, when it was used only by tests and may lead to confusions...